### DISCUSSION TITLE
Tuya Zigbee Water Leak sensor

### DISCUSSION THREAD

**Discussion by sh00t2kill** [Koenkk/zigbee2mqtt#6431] [Devices]
Hi all.
Ive just had a couple of Tuya leak sensors turn up. I bought them ages ago and I think they got lost somewhere in transit, I had totally forogtten about it.

I cant get either of them to even pair.
`Zigbee2MQTT:error 2021-02-24 20:53:57: Failed to interview '0x847127fffe19ce80', device has not successfully been paired`

I cant find any model details anywhere, but these appear to be the same as these:
[https://www.amazon.com/Topuality-Wireless-Flooding-Detector-Detection/dp/B08HGND7KX](url)

Has anyone else had any luck with these devices? Did I just get unlucky?

**Comment by sh00t2kill** [comment]
I contacted the supplier who suggested that it uses "zigbee encyption and only works with the tuya zigbee hub"

**Comment by c0mplex1** [comment]
Hi,

I use this TuYa Leak Sensor. No issues. And it pair immediately.
https://nl.aliexpress.com/item/4001358447417.html
